### PR TITLE
[IMP] mass_mailing: minor improvement in views

### PR DESCRIPTION
**Pull request by fbarbe00**
Purpose
=======
Improve UI of mass_mailing module to give the user a clearer view of the
mailings they send.

Specifications
=============
Remove border and padding around the chatter in mass mailing form view,
add sum and average in mailing list view for KPIs, change sort order
from sent_date to calendar_date, move user_id in form view.

Task-2679537

### CODE REVIEW

File: addons/mass_mailing/models/res_config_settings.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 class ResConfigSettings(models.TransientModel):
     _inherit = 'res.config.settings'
 
-    group_mass_mailing_campaign = fields.Boolean(string="Mailing Campaigns", implied_group='mass_mailing.group_mass_mailing_campaign', help="""This is useful if your marketing campaigns are composed of several emails""")
+    group_mass_mailing_campaign = fields.Boolean(string="Mailing Campaigns", implied_group='mass_mailing.group_mass_mailing_campaign', help="""This is useful if your marketing campaigns are composed of several emails""", default=True)
```
Comment (std-odoo) [inline addons/mass_mailing/models/res_config_settings.py:10]: is the default=True needed ?

File: addons/mass_mailing/security/mass_mailing_security.xml [hunk 2]
```diff
@@ -20,6 +20,10 @@ professional emails and reuse templates.</field>
     </record>
 
     <data noupdate="1">
+        <record model="res.groups" id="base.group_user">
```
Comment (std-odoo) [inline addons/mass_mailing/security/mass_mailing_security.xml:23]: guidelines: id before model

**Review by std-odoo — COMMENTED** [review]
Validation :)

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Pull request closed without merging**